packages/server/src/fhir/repo.ts

```diff
@@ -107,11 +107,11 @@ export interface RepositoryContext {
   remoteAddress?: string;
 
   /**
-   * The current project reference.
-   * This should be the ID/UUID of the current project.
+   * Projects that the Repository is allowed to access.
+   * This should include the ID/UUID of the current project, but may also include other accessory Projects.
```

Comment:
We may want to extend this comment, because there's a fair amount of hidden complexity here.

1. `projects` can be undefined or empty, which is only allowed for system user (and super admins?)
2. `projects` can be length 1, indicating the simple classic case of a user without any linked projects
3. `projects` can have additional length, indicating a primary project and additional linked projects

In any case where `projects.length > 0`, the first project is the "primary" project?